### ISSUE TITLE
Using App Inspection in Android Studio

### ISSUE THREAD

**Issue by max-b** [GrapheneOS/os-issue-tracker#1370]
I've been trying to do some dev work w/ a google pixel 5a running GrapheneOS v2022080500. It's going pretty well, I can run dev versions of the app from android studio and I can run it in the debugger with breakpoints and all :tada: 

The one thing that doesn't seem to _just work_ :tm: is the "App Inspection". I'm trying to observe some worker behavior and though I _can_  see the running process in the dropdown selector, nothing comes up in the "App Inspection" window.

It's not really that big of a deal, just thought I'd ask. Thanks!!!!
![graphene-app-inspector](https://user-images.githubusercontent.com/2491841/184517326-b8533c24-f729-4fe4-a492-b031eacd8084.png)

**Comment by thestinger** [comment]
Can you try turning off exec spawning and rebooting?

**Comment by max-b** [comment]
Yup turned it off and rebooted and does not seem to make a difference

**Comment by Nimmidev** [comment]
I am having the same issue on the latest versions of Android Studio and GrapheneOS (Stable Channel) on a Pixel 6 Pro.

Turning of exec spawning and rebooting also did not change anything for me.

**Comment by Xstoudi** [comment]
Same here, latest Android Studio and GrapheneOS stable.
